src/main/java/net/pms/PMS.java

```diff
@@ -1086,9 +1085,7 @@ public static void main(String args[]) {
 				if (System.getProperty(NOCONSOLE) == null) {
 					System.setProperty(CONSOLE, Boolean.toString(true));
 				}
-			} else {
-				headless = false;
```

Comment:
@SubJunk The author of PMS.main() had made a local variable ```headless``` probably to avoid calling ```isHeadless()``` unnecessary. It was set on line 1090 based on the result of ```isHeadless``` and then read once in line 1100: 
```java
		if (!headless && displayProfileChooser) {
			ProfileChooser.display();
		}
```
 Since ```isHeadless()``` is now rewritten so that it only makes the test once, and after that just returns the previously estabilished status, I simply replaced the test at line 1100 (now 1097) to use isHeadless() instead:
```java
		if (!isHeadless() && displayProfileChooser) {
			ProfileChooser.display();
		}
```
I then deleted the variable and its remaining reference, the line above.